src/main/java/seedu/address/model/person/exceptions/DuplicatePersonException.java

```diff
@@ -1,11 +1,11 @@
 package seedu.address.model.person.exceptions;
```

Comment:
This class can also be deleted.

Reply:
resolving, deleting the class will introduce breaking changes